lib/cf/dashboard/package.json

```diff
@@ -0,0 +1,101 @@
+{
+	"name": "abacus-service-dashboard",
+	"description": "CF Service Dashboard for Abacus",
+	"license": "Apache-2.0",
+	"version": "0.0.1",
+	"private": true,
+	"homepage": "https://github.com/cloudfoundry-incubator/cf-abacus-broker/lib/cf/dashboard",
+	"bugs": {
+		"url": "https://github.com/cloudfoundry-incubator/cf-abacus-broker/issues"
+	},
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/cloudfoundry-incubator/cf-abacus-broker.git"
+	},
+	"keywords": [
+		"cf",
+		"abacus",
+		"dashboard"
+	],
+	"files": [
+		".apprc",
+		"manifest.yml",
+		"src/"
+	],
+	"main": "src/server.js",
+	"scripts": {
+		"start": "node src/server.js --harmony",
+		"stop": "webapp stop",
+		"lint": "eslint",
+		"pub": "publish",
+		"cfpack": "cfpack",
+		"cfpush": "cfpush",
+		"postinstall": "bower install",
+		"jsdoc": "jsdoc -c jsdoc.json",
+		"jshint": "jshint --exclude coverage,docs,jsdoc,node_modules,webapp,test,tmp  --reporter=node_modules/jshint-stylish .",
+		"jshint-ci": "jshint --exclude coverage,docs,jsdoc,node_modules,public,tmp --reporter=checkstyle .",
+		"watch": "mocha src/test/backend --watch",
```

Comment:
We should support this feature in general manner for all modules in Abacus & broker. Filed https://github.com/cloudfoundry-incubator/cf-abacus/issues/633